services/web/server/src/simcore_service_webserver/functions/_controller/_functions_rest_schemas.py

```diff
@@ -79,4 +79,13 @@ class FunctionsListQueryParams(
 ): ...
 
 
+class FunctionDeleteQueryParams(BaseModel):
+    force: Annotated[
+        bool,
+        Field(
+            description="If true, deletes the function even if it has associated jobs.",
```

Comment:
MINOR: add in description " otherwise it errors with HTTP_409_CONFLICT if it has associated jobs"